Dockerfile

```diff
@@ -39,6 +47,8 @@ COPY --from=builder /workspace/manager-${TARGETARCH} /usr/local/bin/manager
 
 COPY entrypoint.sh /usr/local/bin/
 
+VOLUME ["/workspace/data"]
```

Comment:
This is only for e2e test, we should not keep it in normal dockerfile. I suggest we create a new dockerfile for e2e test